### PR TITLE
MINOR:Start processor inside verify message

### PR DESCRIPTION
**Pull request by bbejeck**
This PR fixes a flaky system test.  

I ran six runs of branch builder, and each run was parameterized to repeat the test 25 times for a total of 150 runs.  All test runs passed.

1. https://jenkins.confluent.io/job/system-test-kafka-branch-builder/2122/
2. https://jenkins.confluent.io/job/system-test-kafka-branch-builder/2123/
3. https://jenkins.confluent.io/job/system-test-kafka-branch-builder/2124/
4. https://jenkins.confluent.io/job/system-test-kafka-branch-builder/2128/
5. https://jenkins.confluent.io/job/system-test-kafka-branch-builder/2129/
6. https://jenkins.confluent.io/job/system-test-kafka-branch-builder/2130/

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: streams/src/test/java/org/apache/kafka/streams/tests/StreamsNamedRepartitionTest.java [hunk 1]
```diff
@@ -63,6 +63,7 @@ public static void main(final String[] args) throws Exception {
         final StreamsBuilder builder = new StreamsBuilder();
 
         final KStream<String, String> sourceStream = builder.stream(inputTopic, Consumed.with(Serdes.String(), Serdes.String()));
+        sourceStream.peek((k,v) -> System.out.println(String.format("input data key=%s, value=%s", k, v)));
```
Comment (bbejeck) [inline streams/src/test/java/org/apache/kafka/streams/tests/StreamsNamedRepartitionTest.java]: Used for debugging I'd prefer to leave in should the test have problems in the future

File: tests/kafkatest/tests/streams/streams_named_repartition_topic_test.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def __init__(self, test_context):
         }
 
         self.zookeeper = ZookeeperService(self.test_context, num_nodes=1)
-        self.kafka = KafkaService(self.test_context, num_nodes=1,
+        self.kafka = KafkaService(self.test_context, num_nodes=3,
```
Comment (bbejeck) [inline tests/kafkatest/tests/streams/streams_named_repartition_topic_test.py:44]: Changed this value to be consistent with other system tests

File: tests/kafkatest/tests/streams/streams_named_repartition_topic_test.py [hunk 3]
```diff
@@ -66,7 +66,6 @@ def test_upgrade_topology_with_named_repartition_topic(self):
         for processor in processors:
             processor.CLEAN_NODE_ENABLED = False
             self.set_topics(processor)
-            processor.start()
```
Comment (bbejeck) [inline tests/kafkatest/tests/streams/streams_named_repartition_topic_test.py:69]: This is the fix removed start here and below

File: tests/kafkatest/tests/streams/streams_named_repartition_topic_test.py [hunk 4]
```diff
@@ -89,7 +87,9 @@ def test_upgrade_topology_with_named_repartition_topic(self):
 
     @staticmethod
     def verify_running(processor, message):
-        with processor.node.account.monitor_log(processor.STDOUT_FILE) as monitor:
+        node = processor.node
+        with node.account.monitor_log(processor.STDOUT_FILE) as monitor:
+            processor.start()
```
Comment (bbejeck) [inline tests/kafkatest/tests/streams/streams_named_repartition_topic_test.py:92]: This is the other part of the fix moved processor start here

**Comment by bbejeck** [comment]
ping @guozhangwang, @mjsax, and @vvcephei for reviews

**Review by mjsax — APPROVED** [review]
LGTM

File: tests/kafkatest/tests/streams/streams_named_repartition_topic_test.py (hunk 4 above)
Comment (vvcephei) [reply tests/kafkatest/tests/streams/streams_named_repartition_topic_test.py:92]: What's the significance of after opening the monitor? Does it block on the log file somehow?

Comment (bbejeck) [reply tests/kafkatest/tests/streams/streams_named_repartition_topic_test.py:92]: This closes a very small timing window, I've made similar changes in other system tests before

**Comment by guozhangwang** [comment]
LGTM! Merged to trunk.

File: tests/kafkatest/tests/streams/streams_named_repartition_topic_test.py (hunk 4 above)
Comment (mjsax) [reply tests/kafkatest/tests/streams/streams_named_repartition_topic_test.py:92]: @vvcephei If get don't get the monitor first, we might miss some output lines from the file in `wait_until` below and thus time out even if the correct output was written.

Comment (vvcephei) [reply tests/kafkatest/tests/streams/streams_named_repartition_topic_test.py:92]: :+1:

**Pull request merged**